### PR TITLE
[30] allow test user to copy out XMLrunner reports

### PR DESCRIPTION
**Pull request by d-w-moore**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,8 +13,6 @@ The following database Docker image tags are currently supported:
 
 To add support for a new OS platform/version or database type/version, simply add a new Compose project like those under `projects` and point the scripts at it.
 
-NOTE: iRODS 4.2.x is the only supported series for testing at this time. 4.3.x and others will be added in the future.
```
Comment (alanking) [inline README.md:16]: Is this PR meant to be for 4-2-stable or main?

**Review by alanking — COMMENTED** [review]
Besides the possible branch mix-up, this looks good to me

**Comment by d-w-moore** [comment]
@alanking Oops, I actually did that. It's meant for 4-2-stable of course, then I can cherry pick to main later.  Is there a way we can move this PR to the right place,  @trel ?

**Comment by alanking** [comment]
I think if you just rebase on main here and remove the README change, it will apply cleanly...

Alternatively, you can close this PR and open a new one for main after it's been merged and cherry-picked.

Up to you.

**Comment by trel** [comment]
i think i can redirect a PR... hold please...

**Comment by trel** [comment]
ooh neat.  i think it worked.

File: README.md (hunk 1 above)
Comment (trel) [reply README.md]: redirected the PR, this is moot.

**Comment by alanking** [comment]
Sweet, thanks! # em

**Comment by d-w-moore** [comment]
Ok, #'d and a little explanation added to the commit as well.

**Comment by alanking** [comment]
Thanks. Please cherry-pick to main

**Comment by trel** [comment]
> Ok, #'d and a little explanation added to the commit as well.

in general, try to avoid this... so we have agreement on what is in a commit before it gets #'d.

**Comment by d-w-moore** [comment]
Will do in the future.  Wasn't trying  to sneak anything in but thought it should be said.

**Comment by trel** [comment]
yep, got it.

**Pull request merged**